src/directives/leaflet.js

```diff
@@ -24,9 +23,9 @@ angular.module("leaflet-directive", []).directive('leaflet',
         transclude: true,
         template: '<div class="angular-leaflet-map"><div ng-transclude></div></div>',
         controller: function ($scope) {
-            _leafletMap = $q.defer();
+            this._leafletMap = $q.defer();
```

Comment:
How is this exactly different then being a  closure var? You state that all angular properties of this object will be cleaned afterwards? Why not just clear the var in line 141? Can you point me to the angular documentation or code that does this?